plugins/github-actions/src/api/GithubActionsClient.ts

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { GithubActionsApi } from './GithubActionsApi';
+import { Build, BuildDetails, BuildStatus, WorkflowRun } from './types';
+
+const statusToBuildStatus: { [status: string]: BuildStatus } = {
+  success: BuildStatus.Success,
+  failure: BuildStatus.Failure,
+  pending: BuildStatus.Pending,
+  running: BuildStatus.Running,
+  in_progress: BuildStatus.Running,
+  completed: BuildStatus.Success,
+};
+
+const conclusionToStatus = (conslusion: string): BuildStatus =>
+  statusToBuildStatus[conslusion] ?? BuildStatus.Null;
+
+export class GithubActionsClient implements GithubActionsApi {
+  async listBuilds({
+    owner,
+    repo,
+    token,
+  }: {
+    owner: string;
+    repo: string;
+    token: string;
+  }): Promise<Build[]> {
+    const url = `https://api.github.com/repos/${owner}/${repo}/actions/runs`;
+
+    const response = await fetch(url, {
+      headers: new Headers({
+        Authorization: `Bearer ${token}`,
+      }),
+    });
+
+    if (response.status > 200) {
```

Comment:
`>400`? 2xx are success codes.

Reply:
good catch!

Reply:
Should in general just use `ok`: https://developer.mozilla.org/en-US/docs/Web/API/Response/ok